okta_rules/okta_admin_role_assigned.py

```diff
@@ -1,27 +1,36 @@
 import re
-from panther_base_helpers import okta_alert_context
+from panther_base_helpers import deep_get, okta_alert_context  # pylint: disable=import-error
 
 
 def rule(event):
-    return (
-        event['eventType'] == 'user.account.privilege.grant' and
-        event['outcome'].get('result', '') == 'SUCCESS' and bool(
-            re.search(
-                r'[aA]dministrator',
-                event['debugContext']['debugData'].get('privilegeGranted'))))
+    return (event['eventType'] == 'user.account.privilege.grant' and
+            deep_get(event, 'outcome', 'result') == 'SUCCESS' and bool(
+                re.search(
+                    r'[aA]dministrator',
+                    deep_get(event, 'debugContext', 'debugData',
+                             'privilegeGranted'))))
```

Comment:
`deep_get` returns `None` if the keys are missing, and `re.search(r'[aA]dministrator', None)` throws an exception. We probably want to provide a sensible default here like an empty string or something, unless we know these keys will always be present (in which case I'd imagine `deep_get` is not needed).

Reply:
Good catch! Updated